inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/HttpManager.java

```diff
@@ -84,6 +86,13 @@ public String doSentPost(String url, Object dto) {
         return null;
     }
 
+    public static String toJsonStr(Object obj) {
+        GsonBuilder gsonBuilder = new GsonBuilder();
```

Comment:
The instance of `Gson` should be static and final, as it is a thread-safe class.

Reply:
done